src/cattr/converters.py

```diff
@@ -433,3 +441,21 @@ def _get_dis_func(self, union):
                 "currently. Register a loads hook manually."
             )
         return create_uniq_field_dis_func(*union_types)
+
+    def ib(self, default=attr.NOTHING, validator=None, repr=True, cmp=True,
+           hash=None, init=True, convert=None, metadata=None, type=None,
+           converter=None, factory=None, kw_only=False, src_key=None):
```

Comment:
Instead of the name `src_key`, we could also use something else. `Related` uses [`key`](https://github.com/genomoncology/related/blob/master/tests/ex05_field_names/test_renamed.py). `Marshmallow` uses [`data_key`](https://marshmallow.readthedocs.io/en/3.0/quickstart.html#specifying-serialization-deserialization-keys).

Reply:
I am absolutely agnostic about the name. Maybe `key` would actually be a better choice as we are mapping from a dictionary `key` to the property and back to the `key`.

I'll change it to be that.